### DISCUSSION TITLE
Grafana Dashboard Question

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#2608] [Unsupported Versions] [answered]
In my lab environment, I have build SO 2.3.21 with in a distributed config.
1 - Manager
1 - Storage
1 - Forward
1 - Fleet

As I built each node, a dashboard was created  for most everything. However, a dashboard was not created for the Fleet server. I double checked and all four servers show up in the SOC console as being online.

Will a dashboard be developed for Fleet servers?  

If I build a dashboard for it, will it be deleted when the container get's updated?

Thank you

**Comment by TOoSmOotH** [comment, marked as answer]
A dashboard was never built for it. As long as you create new dashboards they will not get overwritten. If you modify existing then they will. Let me know if you are willing to share a fleet dashboard. :)
